src/pods/canvas/canvas.business.ts

```diff
@@ -73,3 +79,75 @@ export const calculateRelationXCoordinate = (
   xOrigin: calculateRelationXCoordinateOrigin(tableOrigin, tableDestination),
   xDestination: calculateRelationXCoordinateEnd(tableOrigin, tableDestination),
 });
+
+const seekField = (
+  fieldId: GUID,
+  YPosition: number,
+  fields: FieldVm[]
+): seekResult => {
+  let found = false;
+  let parentCollapsed = false;
+  let newYPosition = YPosition;
+
+  for (let i = 0; i < fields.length && !found; i++) {
+    const field = fields[i];
+    if (field.id === fieldId) {
+      found = true;
+      return {
+        found,
+        parentCollapsed,
+        YPosition: newYPosition,
+      };
```

Comment:
We could create a helper function

```tsx
const fieldFound = (parentCollapsed: boolean, YPosition: number) => ({
  found: true,
  parentCollapsed,
  YPosition
});
```

Then we can replace and simplify:

```diff
if (field.id === fieldId) {
-      found = true;
- 
-     return {
-        found,
-        parentCollapsed,
-       YPosition: newYPosition,
-    };
+  return fieldFound(parentCollapsed, YNewPosition);
}
```

Reply:
It's fixed